.github/workflows/cd.yml

```diff
@@ -0,0 +1,60 @@
+name: CD
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+  release:
+    types:
+      - published
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  # Many color libraries just need this to be set to any value, but at least
+  # one distinguishes color depth, where "3" -> "256-bit color".
+  FORCE_COLOR: 3
+
+jobs:
+  dist:
+    name: Distribution build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: hynek/build-and-inspect-python-package@v2
+
+  publish:
+    needs: [dist]
+    name: Publish to PyPI
+    environment: pypi
+    permissions:
+      id-token: write
+      attestations: write
+      contents: read
+    runs-on: ubuntu-latest
+    if: github.event_name == 'release' && github.event.action == 'published'
+
+    steps:
+      - uses: actions/download-artifact@v4
+        with:
+          name: Packages
+          path: dist
+
+      - name: Generate artifact attestation for sdist and wheel
+        uses: actions/attest-build-provenance@v1.4.1
+        with:
+          subject-path: "dist/*"
+
+      - uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          # Remember to tell (test-)pypi about this repo before publishing
+          # Remove this line to publish to PyPI
```

Comment:
Don't forget to do this?

Reply:
Still need to set up the publishing workflows, and discuss PyPI and Conda names for the package (virtual_ship?)

Reply:
Yes, I think `virtual_ship` (or `virtualship`?) is a good name for the package. Since it doesn't need to be used in a classroom setting, I would not add `classroom` to the name

Reply:
`virtualship` has my vote

Reply:
Agree, and doesn't exist (yet) on https://pypi.org/search/?q=virtualship nor https://anaconda.org/search?q=virtualship

Reply:
Or even better (IMO) would be `VirtualShipParcels`. That is also how @ammedd calls the package in her draft manuscript

Reply:
We need to adhere to the [package naming conventions](https://docs.conda.io/projects/conda-build/en/latest/concepts/package-naming-conv.html) so `virtual-ship-parcels`* or `virtual_ship_parcels` or `virtualshipparcels` would be better.

*my vote of the three

Reply:
OK, `virtual-ship-parcels` is a good choice then

Reply:
Just confirming, for the package name itself, stick to `virtualship`? i.e.
- `conda install virtual-ship-parcels`
- `import virtualship`

Its not uncommon for the install name to be different from the import name. And should the repo name be reconsidered as well?

Reply:
Yes, or `import virtual-ship-parcels as vsp`? 

And yes, the name of this repo should then also be changed. But let's first see what @ammedd prefers

Reply:
> Yes, or `import virtual-ship-parcels as vsp`?

Python syntax would dictate that we use underscores instead of `-`. Perhaps `virtualship` for bot the install and the import is cleaner, and then use `VirtualShipParcels` in documentation and readmes

Reply:
- Conda/pypi/repo: virtualship
- full name: VirtualShipParcels